subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -113,15 +133,24 @@ static void prio_recv_thread(void *p1, void *p2, void *p3)
 			/* Find out and store the class for this node */
 			node_rx->hdr.user_meta = hci_get_class(node_rx);
 
-			buf = process_prio_evt(node_rx);
+			buf = process_prio_evt(node_rx, &evt_is_normal);
```

Comment:
returning a bt_recv_type and handling the call to bt_recv_prio or bt_recv in a switch case is cleaner.

Reply:
@carlescufi @jhedberg opinions on removing bt_recv_is_prio and and bt_recv_is_normal with a bt_recv_type as suggested?
I prefer my way, but not a strong opinion.
Related comment: https://github.com/zephyrproject-rtos/zephyr/pull/25954#discussion_r435038613

Reply:
I prefer if (is_normal) over if (type == normal || type == both)

Reply:
I prefer a bitmask, see my previous comment.
@jhedberg feedback welcome here as well.